src/cli/remove.rs

```diff
@@ -74,8 +105,12 @@ pub async fn execute(args: Args) -> miette::Result<()> {
     }
 
     for result in &results {
-        if let Err(e) = result {
-            eprintln!("{e}");
+        match result {
+            DependencyRemovalResult::PixiDeps(Err(e))
+            | DependencyRemovalResult::PyPiDeps(Err(e)) => {
+                eprintln!("{e}");
+            }
+            _ => {}
```

Comment:
We will never get here because you unwrap the result a few lines higher.

Reply:
that was reworked, please take a look